conf/atac/preprocess.config

```diff
@@ -0,0 +1,17 @@
+params {
+    sc {
+        atac{
+            preprocess {
+                metadata = 'metadata.tsv'
+            }
+        }
+    }
+}
+
+//includeConfig './../../src/snaptools/snaptools.config'
+includeConfig './../../src/singlecelltoolkit/singlecelltoolkit.config'
+includeConfig './../../src/trimgalore/trimgalore.config'
+includeConfig './../../src/bwamaptools/bwamaptools.config'
+includeConfig './../../src/sinto/sinto.config'
+includeConfig './../../src/bap/bap.config'
```

Comment:
I would not scope all these tools under `atac` since some are more generic tools like `bwa`, `trimgalore` because we want to use these later for something else it's going to be weird to have it under `atac` in the config.

Reply:
I'm not sure I understand. I created the `atac/preprocess.config` for a specific workflow, and it imports all of these tools. If you needed any of these tools for other purposes later, you can run these `includeConfig ...` commands to import them in another config that would not have anything to do with atac.

Reply:
I was referring to the `atac` scope in those configs.
But this is resolved since you rescoped the `params`